packages/neuron-wallet/src/utils/hex.ts

```diff
@@ -0,0 +1,9 @@
+export default class HexUtils {
+  public static toDecimal(hex: string): string {
```

Comment:
It should be named as `toInt`

Reply:
It's a convert between `decimal` and `hex`, not `Integer` and  `String`